go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	k8s.io/metrics v0.32.2
 	k8s.io/utils v0.0.0-20241104100929-3ea5e8cea738
 	sigs.k8s.io/controller-runtime v0.19.4
-	sigs.k8s.io/jobset v0.7.3
+	sigs.k8s.io/jobset v0.8.0-devel
```

Comment:
```suggestion
	sigs.k8s.io/jobset v0.8.0
```

Reply:
This is follow-up for this comment https://github.com/kubernetes-sigs/kueue/pull/4455#issuecomment-2694039179.

Reply:
Yeah, I hoped we could freeze controller-runtime at older version, but not sure this is supported in go.mod. 
If not supported, we don't want to rely on the -devel. I guess we need to wait for the kuberay bump.